tests/integration/640.command.recipes.test.cjs

```diff
@@ -96,3 +97,46 @@ test('Does not generate a new VS Code settings file if the user does not confirm
     t.is(error.code, 'ENOENT')
   })
 })
+
+test.only('Handles JSON with comments', async (t) => {
+  await withSiteBuilder('repo', async (builder) => {
+    const comment = '// sets value for someSetting'
+    await builder
+      .withContentFile({
+        path: '.vscode/settings.json',
+        content: `{
+          "deno.enablePaths":["/some/path"],
+          "someSetting":"value" ${comment}
+        }`,
+      })
+      .buildAsync()
+
+    const childProcess = execa(cliPath, ['recipes', 'vscode'], {
+      cwd: builder.directory,
+    })
+    const settingsPath = resolve(builder.directory, '.vscode', 'settings.json')
+
+    handleQuestions(childProcess, [
+      {
+        question: `There is a VS Code settings file at ${settingsPath}. Can we update it?`,
+        answer: answerWithValue(CONFIRM),
+      },
+      {
+        question: 'The Deno VS Code extension is recommended. Would you like to install it now?',
+        answer: answerWithValue(NO),
+      },
+    ])
```

Comment:
@danez Note here: 'The Deno VS Code extension is recommended. Would you like to install it now?' is asked on my local system but it seems like this question is not asked during the CI run. The test just ends up stuck & didn't run without handling this question.

(This is applicable for other tests in this file too.)